content/workshops/openshift_4_101/lab2-byocontainer.md

```diff
@@ -74,14 +79,23 @@ Select the option for "Image name from external registry" and enter "sonatype/ne
 
 <blockquote>
 Click on the pop-up menu underneath "Application", and select "Unassigned", to have the application not be in the group with the Butterfly Terminal.
```

Comment:
To remove the application display from the Butterfly Terminal group, select "Unassigned" from the pop-up menu.